src/gui/systray.h

```diff
@@ -29,6 +29,7 @@ namespace OCC {
 #ifdef Q_OS_OSX
 bool canOsXSendUserNotification();
 void sendOsXUserNotification(const QString &title, const QString &message);
+void setTrayWindowLevelAndVisibleOnAllSpaces(QQuickWindow *qtWindow);
```

Comment:
I'd rename the parameter from qtWindow to window. Also seeing how it's used please change the type to QWindow.